presto-verifier/src/main/java/com/facebook/presto/verifier/framework/VerifierConfig.java

```diff
@@ -407,4 +410,21 @@ public VerifierConfig setRunningMode(String runningMode)
         }
         return this;
     }
+
+    public Optional<String> getFunctionSubstitutes()
+    {
+        return functionSubstitutes;
+    }
+
+    @ConfigDescription("Specification of function substitutions, in the format of" +
+            " /foo(c0,_)/bar(c0)/,/fred(c0,c1)/baz(qux(c1,c0))/,/foobar(c0)/if(qux(c1),bar(c0),baz(c1))/,...," +
+            " where foo(c0, _) would be substituted by bar(c0), with the declared arguments applied" +
+            " to the corresponding positions. Concatenate function substitutions with comma.")
+    @Config("function-substitutes")
+    public VerifierConfig setFunctionSubstitutes(String functionSubstitutes)
```

Comment:
You also need to update `TestVerifierConfig` with this property

Reply:
Change to this class actually belongs to the new PR. I will move it there and address the comment there. Had put it in the wrong PR when I split the change.